src/test_typing_extensions.py

```diff
@@ -5351,25 +5379,56 @@ def test_invalid_uses(self):
 
         with self.assertRaisesRegex(
             TypeError,
-            'The last parameter to Concatenate should be a ParamSpec variable',
+            'The last parameter to Concatenate should be a ParamSpec variable or ellipsis',
         ):
             Concatenate[P, T]
 
+        # Cannot construct a Callable with Ellipsis in 3.8 as args must be types
+        if sys.version_info[:2] >= (3, 9, 2):
+            with self.assertRaisesRegex(
+                TypeError,
+                'is not a generic class',
+            ):
+                Callable[Concatenate[int, ...], Any][T]
+
         if not TYPING_3_11_0:
             with self.assertRaisesRegex(
                 TypeError,
                 'each arg must be a type',
             ):
                 Concatenate[1, P]
 
+            with self.assertRaisesRegex(
+                TypeError,
+                'each arg must be a type.',
+            ):
+                Concatenate[1, ..., P]
+
+    @skipUnless(TYPING_3_11_0, "Cannot be backported to <=3.9"
+                               "Cannot use ... with 3.10 typing._ConcatenateGenericAlias")
+    def test_alias(self):
+        P = ParamSpec("P")
+        C1 = Callable[Concatenate[int, P], Any]
+        # Python <= 3.9 fails because parameters to generic types must be types.
+        # For Python 3.10 & typing._ConcatenateGenericAlias will
```

Comment:
This sentence seems garbled, I'm not sure what you're trying to say.

Reply:
Removed this block.

My idea was to note about these three cases

1. Cannot use with <= 3.9 because types need to be passed
2. Cannot use `typing_extensions.Concatenate`, `_concatenate_getitem`  with the original `typing._ConcatenateGenericAlias`
3. A full backport to `typing_extensions._ConcatenateGenericAlias` might be possible but is not implemented.

---

Btw. should the function be renamed? I was not sure how to name it and now realized test_alias is used elsewhere with different semantics.

Reply:
I've now renamed the function I named `test_alias` before.